var/spack/repos/builtin/packages/py-python-ldap/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPythonLdap(PythonPackage):
+    """python-ldap provides an object-oriented API to access LDAP directory
+    servers from Python programs.
+    """
+
+    homepage = "https://www.python-ldap.org/en/python-ldap-3.2.0/"
+    url      = "https://pypi.io/packages/source/p/python-ldap/python-ldap-3.2.0.tar.gz"
+
+    version('3.2.0', sha256='7d1c4b15375a533564aad3d3deade789221e450052b21ebb9720fb822eccdb8e')
+    version('3.0.0', sha256='86746b912a2cd37a54b06c694f021b0c8556d4caeab75ef50435ada152e2fbe1')
+
+    depends_on('openldap+client_only', type=('build', 'link', 'run'))
```

Comment:
Can it not be built with `openldap~client_only`?

Reply:
Not sure, but I think the intention of this package is to be able to manage or interact with an openldap installation on a server. Hence, client_only installation of openldap as suggested by developers should be sufficient. If you think that it's a good idea, I can create a client_only variant (enabled by default) triggering a client_only building of openldap.

Reply:
Nah, that's okay. I'll defer to the developers if they recommend the client-only build.